device/celestica/x86_64-cel_questone_2-r0/Questone_2/port_config.ini

```diff
@@ -0,0 +1,57 @@
+# name      lanes            alias   index  speed   mtu   admin_status  fec
+Ethernet1   49               25GE1   0      25000   9216  up            rs
```

Comment:
According to this doc, alias naming should follow `etpN` where N>=1

Reply:
Hi, Blueve, thanks for you review. Is this necessary? It's just an alias. I saw different manufacturers have different  alias styles.